api/camunda/camunda-openapi.yaml

```diff
@@ -692,6 +731,52 @@ paths:
           description: >
             The decision requirements search query failed.
             More details are provided in the response body.
+  /authorizations:
+    post:
+      tags:
+        - Authorization
+      summary: Create an authorization
```

Comment:
change-at-the-source, descriptions, new:

This endpoint is missing a `description` (in this case, it's used as the main paragraph content on the page, and since there isn't one, it falls back to the `summary`. It's noticeable because it isn't formatted as a sentence.)

Reply:
owner: me

Reply:
Nudged in https://github.com/camunda/camunda-docs/pull/4217#discussion_r1746202818